ugs-platform/ugs-platform-ugscore/src/main/resources/resources/icons/export.sh

```diff
@@ -4,7 +4,20 @@ if [ -f /Applications/Inkscape.app/Contents/Resources/bin/inkscape ]; then
 	alias inkscape="/Applications/Inkscape.app/Contents/Resources/bin/inkscape"
 fi
 
+# Make sure nobody is using black instead of blackish
```

Comment:
@breiler this PR is basically impossible to review now, I tweaked the export script to generate the _dark and _disabled_dark files. It does make the connect icon a little weird though, it will now have a teal border in dark mode since dark/_disabled_dark versions will be created.

Reply:
@dastultz I noticed that some icons used black and some used a very dark gray. Is this intentional? I converted everything to use the same dark gray color, the colors can be updated pretty easily now if we want to experiment.

Reply:
I'm responsible for the dark gray color which is stolen from MacOSX. I wanted it to be more consistent with the dark gray panel and the menu text colors. If you don't approve I can change it back to black.

Reply:
They both look black to me, I was just trying to make it consistent.